server/tests/controller/article.spec.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-console */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '<server>/app';
+import models from '<server>/models';
+import { article } from '<fixtures>/article';
```

Comment:
i think it'll be better to use the newArticle object instead of creating a new one